File: sqlUpdate.py
import pymysql

# Open database connection
db = pymysql.connect("localhost", "root", "password", "school")

# prepare a cursor object using cursor() method
cur = db.cursor(pymysql.cursors.DictCursor)

sql = "UPDATE students SET age=20 WHERE name='Junhong'"

# execute SQL query using execute() method.
cur.execute(sql)

# commit changes in database
db.commit()

# disconnect from server
cur.close()
db.close()

File: sqlUpdate.php
<?php
$servername = "localhost";
$username = "root";
$password = "password";
$dbname = "school";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);
// Check connection
if ($conn->connect_error) {
    die("Connection failed: " . $conn->connect_error);
}

$name = "Junhong";

$sql = "UPDATE students SET age=18 WHERE name='$name' AND age=17";
$result = $conn->query($sql);

if ($conn->query($sql) === TRUE) {
    echo "Record updated successfully";
}
else {
    echo "Error: " . $sql . "\n" . $conn->error;
}
$conn->close();
?> 
